### DISCUSSION TITLE
how to train inpainting model with largre pictures

### DISCUSSION THREAD

**Discussion by ncoll** [open-mmlab/mmediting#1185] [Models] [answered]
thank you for your project.first
now i want to train inpainting model DeepFillv2 use my own data,my pictures have various sizes. Will they affect the effect of the model, because my pictures are relatively large(5000*7000). how to deal with larger pictures, thank you for your answer

**Comment by ckkelvinchan** [comment]
@nbei

**Comment by zengyh1900** [comment, marked as answer]
Hi @ncoll, 
Since DeepFillv2 is trained on 256x256, the performance will drop significantly on pictures with relatively larger resolutions. 
If you want to deal with larger pictures, you can resize the images to a smaller size. 

Besides, I would like to suggest another two inpainting works for high-resolution images: 
1. [Contextual Residual Aggregation for Ultra High-Resolution Image Inpainting](http://openaccess.thecvf.com/content_CVPR_2020/html/Yi_Contextual_Residual_Aggregation_for_Ultra_High-Resolution_Image_Inpainting_CVPR_2020_paper.html), Yi et al., CVPR 2020. 
2. [High-Resolution Image Inpainting with Iterative Confidence Feedback and Guided Upsampling](https://arxiv.org/abs/2005.11742), Zeng et al., ECCV 2020. 

I hope it helps! :)
